internal/provider/bootstrap.go

```diff
@@ -197,8 +198,16 @@ func oneTimeBootstrap(l logging.StandardLogger, c *providerConfig.Config, vpnSet
 	}
 
 	if utils.IsOpenRCBased() {
+		// Instead of creating an empty file, let's use the one generated by the k3s installer
+		content, err := os.ReadFile("/etc/init.d/k3s")
```

Comment:
Do we want to error out here? Or set a default that just runs k3s?

Reply:
good question, what is the bare minimum for one of these init files? the read content is as follows:

```
#!/sbin/openrc-run

depend() {
    after network-online
    want cgroups
}

start_pre() {
    rm -f /tmp/k3s.*
}

supervisor=supervise-daemon
name=k3s
command="/usr/bin/k3s"
command_args="server \
    >>/var/log/k3s.log 2>&1"

output_log=/var/log/k3s.log
error_log=/var/log/k3s.log

pidfile="/var/run/k3s.pid"
respawn_delay=5
respawn_max=0

set -o allexport
if [ -f /etc/environment ]; then . /etc/environment; fi
if [ -f /etc/rancher/k3s/k3s.env ]; then . /etc/rancher/k3s/k3s.env; fi
set +o allexport
```

chatGPT says this would be the bare minimum:

```
#!/sbin/openrc-run

depend() {
    after network-online
    want cgroups
}

name=k3s
command="/usr/bin/k3s"
command_args="server >>/var/log/k3s.log 2>&1"

output_log=/var/log/k3s.log
error_log=/var/log/k3s.log

pidfile="/var/run/k3s.pid"
respawn_delay=5
respawn_max=0
```

Reply:
Umm I think we may want to fail hard here as that would mean that we are missing shipped files no?